nodeApps/createPrivateRelease.sh

```diff
@@ -88,17 +112,25 @@ echo "Latest commit is: ${CURRENT_SHA}"
 # Tag this deploy and make a release in github
 RELEASE_JSON="{\"tag_name\": \"v${THIS_VERSION}\",\"target_commitish\": \"${CURRENT_SHA}\",\"name\": \"v${THIS_VERSION}\",\"body\": \"${MESSAGE}\",\"draft\": false,\"prerelease\": false}"
```

Comment:
The link back is there: https://github.com/holidayextras/deployment-helpers/pull/10/files#diff-efc43368f159a2d0209883f8af2537a9R103